scripts/update-index-alias.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+"""
+A script to update the search index an alias is assoiciated with.
+
+To delete old indexes (those with no alias), set --delete-old-indexes=yes
+
+Usage:
+scripts/update-index-alias.py <alias> <target> <stage> <search-api-endpoint> [options]
+
+Options:
+    --delete-old-indexes=<delete-old-indexes>  [default: ]
```

Comment:
Yep. I was trying to get the default to be something 'falsey'. This default gives an empty string which is.
Problem is, if you set it to anything else, like 'no' for example, it's a truthy value. That's why it explicitly checks for 'yes' as a string literal in the code. Perhaps there's a better way of doing it, but setting the default to 'no' won't hurt.